__tests__/lib/string.test.ts

```diff
@@ -852,11 +856,15 @@ describe('parseStarterPackHttpUri', () => {
     null,
     null,
     null,
+    {name: 'did:123', rkey: 'rkey'},
+    null,
+    null,
+    null,
   ]
 
   it('returns the correct name and rkey when input is valid', () => {
     for (let i = 0; i < inputs.length; i++) {
-      const result = parseStarterPackHttpUri(inputs[i])
+      const result = parseStarterPackUri(inputs[i])
```

Comment:
Kinda nit kinda not: should really use `it.each` and a nice interpolated string for this, or refactor into separate test cases like I did for `ago` tests. Reason being, it's hard to know which part failed in tests like this, why tests exist, if duplicate tests exist, etc, esp when the arrays of tests grow.

Reply:
Yea, I've been wanting to refactor a couple of the other tests in this thing for the same reason (that embed test is brutal). Might as clean this one up now.